keras_cv/models/object_detection/yolo_v8/yolo_v8_detector.py

```diff
@@ -609,8 +601,12 @@ def decode_predictions(
 
         return self.prediction_decoder(box_preds, scores)
 
-    def make_predict_function(self, force=False):
-        return predict_utils.make_predict_function(self, force=force)
+    def predict_step(self, *args):
```

Comment:
What's going on here? Do we not need the util anymore?

Reply:
We don't need to use `predict_utils.make_predict_function` because it was a hack we put in to fix XLA compilation with NMS. This is resolved using my new NMS implementation